docs/installation/installation.md

```diff
@@ -1,5 +1,9 @@
 # Installation
 
+!!! note
+
+    GPUStack v2 uses a single unified container image for all GPU device types.
```

Comment:
Put it next to the docker run command. Users have more context about container image there.

Reply:
<img width="1285" height="482" alt="image" src="https://github.com/user-attachments/assets/3a67f3b3-975c-4c6a-914d-a9f922b05789" />

Reply:
I misunderstood.